<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESP8266 NodeMcu GUI</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabs"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_view">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="menu_view_actions"/>
    <addaction name="menu_view_files"/>
    <addaction name="menu_view_log"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="menu_open_file"/>
    <addaction name="menu_open_dir"/>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="menu_prefs"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_settings"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_view_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="menu_prefs">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="menu_view_actions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Actions</string>
   </property>
  </action>
  <action name="menu_view_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Files</string>
   </property>
  </action>
  <action name="menu_open_file">
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menu_open_dir">
   <property name="text">
    <string>Open &amp;Path</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="menu_prefs">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
